core/store/orm/locking_strategies_test.go

```diff
@@ -171,7 +172,6 @@ func TestPostgresLockingStrategy_WhenReacquiredOriginalNodeErrors(t *testing.T)
 	require.NoError(t, err)
 	defer lock.Unlock(delay)
 
-	require.Panics(t, func() {
-		_ = store.ORM.RawDB(func(db *gorm.DB) error { return nil })
-	})
+	_ = store.ORM.RawDB(func(db *gorm.DB) error { return nil })
+	gomega.NewGomegaWithT(t).Eventually(gracefulpanic.Wait()).Should(gomega.BeClosed())
```

Comment:
What's gomega?  It looks like we introducing a third testing dependency into our code base.  It it already being used somewhere?

Reply:
We're already using it elsewhere.

Reply:
Maybe try with testify instead? -> https://godoc.org/github.com/stretchr/testify/assert#Eventually

Reply:
Looks like this is used throughout the codebase